packages/nodes-base/nodes/Okta/UserDescription.ts

```diff
@@ -0,0 +1,524 @@
+				name: 'city',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'State',
+				name: 'state',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Zip Code',
+				name: 'zipCode',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Country Code',
+				name: 'countryCode',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Postal Address',
+				name: 'postalAddress',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Preferred Language',
+				name: 'preferredLanguage',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Locale',
+				name: 'locale',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Timezone',
+				name: 'timezone',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'User Type',
+				name: 'userType',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Employee Number',
+				name: 'employeeNumber',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Cost Center',
+				name: 'costCenter',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Organization',
+				name: 'organization',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Division',
+				name: 'division',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Department',
+				name: 'department',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'ManagerId',
+				name: 'managerId',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Manager',
+				name: 'manager',
+				type: 'string',
+				default: '',
+			},
+		],
+	},
+	// Fields specific to 'create' operation
+	{
+		displayName: 'Activate',
+		name: 'activate',
+		type: 'boolean',
+		displayOptions: {
+			show: {
+				resource: ['user'],
+				operation: ['create'],
+			},
+		},
+		default: false,
+		description: 'Whether to activate the user',
+	},
+	// Fields specific to 'getAll' operation
+	{
+		displayName: 'Search Query',
+		name: 'searchQuery',
+		type: 'string',
+		placeholder: 'Filter Users by email, last name, or first name',
+		displayOptions: {
+			show: {
+				resource: ['user'],
+				operation: ['getAll'],
+			},
+		},
+		default: '',
+		routing: {
+			request: {
+				qs: {
+					prefix: '={{$value}}',
+				},
+			},
+		},
+	},
+	{
+		displayName: 'User ID',
+		name: 'userId',
+		type: 'resourceLocator',
+		default: { mode: 'list', value: '' },
+		required: true,
+		modes: [
+			{
+				displayName: 'From List',
+				name: 'list',
+				type: 'list',
+				placeholder: 'Select a user...',
+				typeOptions: {
+					searchListMethod: 'getUsers',
+					searchable: true,
+				},
+			},
+			{
+				displayName: 'ID',
+				name: 'id',
+				type: 'string',
+				placeholder: '00u1abcd2345EfGHIjk6',
+			},
+		],
+		displayOptions: {
+			show: {
+				resource: ['user'],
+				operation: ['get', 'update', 'delete'],
```

Comment:
When updating a user this should probably be the first field rather than the last

Reply:
done